src/services/StreamLabs.js

```diff
@@ -1,17 +1,26 @@
-const axios = require('axios');
+const axios = require('axios').default;
 
 class StreamLabs {
-	constructor({ token }) {
+	constructor({ token }, logger) {
```

Comment:
We can add the logger as an option in the first parameter to follow [this refactor](https://refactoring.guru/introduce-parameter-object)

Reply:
These args are different, first one could be grouped into a settings object, second one is an injected service/dependency instance. 
They should be separated IMO

Reply:
Will revise when we implement #50